index.js

```diff
@@ -69,7 +69,7 @@ var ReactFileReader = function (_React$Component) {
               files.push(reader.result);
 
               if (files.length === ef.length) {
-                _this.props.handleFiles(files);
+                _this.props.handleFiles(files, ef);
```

Comment:
you don't need to make these changes, you need to compile the app by running npm build and it will use babel to convert the file to vanilla JS.. The only file you should be editing is ReactFileReader.js